app/templates/index.hbs

```diff
@@ -24,40 +24,40 @@
     <div id='stats'>
         <div class='downloads'>
             {{svg-jar "download"}}
-            <span class='num' data-test-total-downloads>{{format-num model.num_downloads}}</span>
+            <span class='num' data-test-total-downloads>{{if hasData (format-num model.num_downloads) "---,---,---"}}</span>
             <span class='desc small'>Downloads</span>
         </div>
         <div class='crates'>
             {{svg-jar "crate"}}
-            <span class='num' data-test-total-crates>{{format-num model.num_crates}}</span>
+            <span class='num' data-test-total-crates>{{if hasData (format-num model.num_crates) "---,---"}}</span>
             <span class='desc small'>Crates in stock</span>
         </div>
     </div>
 </div>
 
 <div id='home-crates' class='crate-lists'>
     <div id='new-crates' data-test-new-crates>
-        <h2>New Crates</h2>
+        <h2>New Crates {{#if dataTask.isRunning}}<img src="/assets/ajax-loader.gif">{{/if}}</h2>
```

Comment:
Could we have a component for the spinner? Like `{{ui-spinner task=dataTask}}`.

Reply:
sure, though I'd keep the `if` condition outside of the component and just wrap the `<img>` element. IMHO this is an independent refactoring though and should not block merging this PR here, right?

Reply:
:+1: